{{cookiecutter.project_name}}/tasks.py

```diff
@@ -64,7 +64,7 @@ def setup_venv(
 
     if not Path(venv_name).exists():
         echo_header(
-            f"{Emo.DO} Creating virtual environment for {Emo.PY}{python_version}",
+            f"{Emo.DO} Creating virtual environment for {python_version}{Emo.PY}",
```

Comment:
I have a few refactors to this thing, but I will create a PR at some point with a suggestion